run_blockchain_terraformer.py

```diff
@@ -103,27 +108,32 @@
 EQUALIZER_V2_NAME = "equalizer_v2"
 SOLIDLY_V2_NAME = "solidly_v2"
 VELODROME_V2_NAME = "velodrome_v2"
+CLEOPATRA_V2_NAME = "cleopatra_v2"
+STRATUM_V2_NAME = "stratum_v2"
 
-SOLIDLY_FORKS = [AERODROME_V2_NAME, VELOCIMETER_V2_NAME, SCALE_V2_NAME, VELODROME_V2_NAME]
+SOLIDLY_FORKS = [AERODROME_V2_NAME, VELOCIMETER_V2_NAME, SCALE_V2_NAME, VELODROME_V2_NAME, CLEOPATRA_V2_NAME, STRATUM_V2_NAME]
 
 
 def get_fee_1(address: str, is_stable: str, factory_contract: Contract):
     return factory_contract.caller.getFee(address)
 
-
 def get_fee_2(address: str, is_stable: str, factory_contract: Contract):
     return factory_contract.caller.getRealFee(address)
 
-
 def get_fee_3(address: str, is_stable: str, factory_contract: Contract):
     return factory_contract.caller.getFee(address, is_stable)
 
+def get_fee_4(address: str, is_stable: str, factory_contract: Contract):
+    return factory_contract.caller.getFee(is_stable)
+
 SOLIDLY_EXCHANGE_INFO = {
     "velocimeter_v2": {"decimals": 5, "factory_abi": VELOCIMETER_V2_FACTORY_ABI, "pool_abi": VELOCIMETER_V2_POOL_ABI, "fee_function": get_fee_1},
     "equalizer_v2": {"decimals": 5, "factory_abi": SCALE_V2_FACTORY_ABI, "pool_abi": EQUALIZER_V2_POOL_ABI, "fee_function": get_fee_2},
     "aerodrome_v2": {"decimals": 5, "factory_abi": SOLIDLY_V2_FACTORY_ABI, "pool_abi": SOLIDLY_V2_POOL_ABI, "fee_function": get_fee_3},
     "velodrome_v2": {"decimals": 5, "factory_abi": SOLIDLY_V2_FACTORY_ABI, "pool_abi": SOLIDLY_V2_POOL_ABI, "fee_function": get_fee_3},
     "scale_v2": {"decimals": 18, "factory_abi": SCALE_V2_FACTORY_ABI, "pool_abi": SOLIDLY_V2_POOL_ABI, "fee_function": get_fee_2},
+    "cleopatra_v2": {"decimals": 5, "factory_abi": SCALE_V2_FACTORY_ABI, "pool_abi": EQUALIZER_V2_POOL_ABI, "fee_function": get_fee_4},
+    "stratum_v2": {"decimals": 5, "factory_abi": VELOCIMETER_V2_FACTORY_ABI, "pool_abi": VELOCIMETER_V2_POOL_ABI, "fee_function": get_fee_1},
```

Comment:
All of this has changed (imported from `solidly_v2.py`, so these changes are required only in that file).

Please pull or rebase from the `develop` branch.